install-agent.sh

```diff
@@ -67,8 +67,21 @@ check_netcat()
 check_omega_agent() {
   if ps ax | grep -v grep | grep "omega-agent" > /dev/null
   then
-    echo "Omega Agent service running.Stop omega-agent"
-    service omega-agent stop > /dev/null 2>&1
+    echo "Omega Agent service is running now... "
+    echo "Wraning!!! Continue installation will overwrite the original version"
```

Comment:
Wraning => Warning

echo "Warning!!! Continue to install will update the current version"